subsys/bluetooth/services/dis.c

```diff
@@ -222,14 +386,83 @@ static int dis_set(const char *name, size_t len_rd,
 		len = read_cb(store, &dis_sw_rev, sizeof(dis_sw_rev) - 1);
 		if (len < 0) {
 			LOG_ERR("Failed to read software revision from storage"
-				       " (err %zd)", len);
+				" (err %zd)",
+				len);
 		} else {
 			dis_sw_rev[len] = '\0';
 
 			LOG_DBG("Software revision set to %s", dis_sw_rev);
 		}
 		return 0;
 	}
+#endif
+#if defined(CONFIG_BT_DIS_UDI)
+	if (!strncmp(name, "udi_label", nlen)) {
+		dis_update_udi_value(store, BT_DIS_UDI_LABEL_STR_REF, read_cb, "label");
+		return 0;
+	}
+
+	if (!strncmp(name, "udi_di", nlen)) {
+		dis_update_udi_value(store, BT_DIS_UDI_DI_STR_REF, read_cb, "device information");
+		return 0;
+	}
+
+	if (!strncmp(name, "udi_issuer", nlen)) {
+		dis_update_udi_value(store, BT_DIS_UDI_ISSUER_STR_REF, read_cb, "issuer");
+		return 0;
+	}
+
+	if (!strncmp(name, "udi_authority", nlen)) {
+		dis_update_udi_value(store, BT_DIS_UDI_AUTHORITY_STR_REF, read_cb, "authority");
+		return 0;
+	}
+#endif
+#if defined(CONFIG_BT_DIS_SYSTEM_ID)
+	if (!strncmp(name, "sysid_oui", nlen)) {
+		uint32_t oui = 0;
+
+		len = read_cb(store, &oui, sizeof(oui));
+		if (len < 0) {
+			LOG_ERR("Failed to read System ID OUI from storage"
+				" (err %zd)",
+				len);
+		} else {
+			sys_put_le24(oui, &dis_system_id[5]);
+			LOG_DBG("System ID OUI set to %06X", oui);
+		}
+		return 0;
+	}
+	if (!strncmp(name, "sysid_identifier", nlen)) {
+		uint64_t identifier = 0;
+
+		len = read_cb(store, &identifier, sizeof(identifier));
+		if (len < 0) {
+			LOG_ERR("Failed to read System ID identifier from storage"
+				" (err %zd)",
+				len);
+		} else {
+			sys_put_le40(identifier, &dis_system_id[0]);
+			LOG_DBG("System ID identifier set to %10llX", identifier);
+		}
+		return 0;
+	}
+#endif
+#if defined(CONFIG_BT_DIS_IEEE_RCDL)
+	if (!strncmp(name, "ieeercdl", nlen)) {
+		len = read_cb(store, &dis_ieee_rcdl, sizeof(dis_ieee_rcdl) - 1);
+		if (len < 0) {
+			LOG_ERR("Failed to read IEEE 11073-20601 Regulatory Certification Data "
+				"List from storage"
+				" (err %zd)",
+				len);
```

Comment:
And here

Reply:
The last two lines here can be merged, but `clang-format` automatically inserts a line break before `len`